source_account/setup_ec2.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+echo "===== Initial Update ====="
+sudo apt-get update
+
+echo "===== Installing packages ====="
+sudo DEBIAN_FRONTEND=noninteractive apt-get install -y \
+  software-properties-common \
+  build-essential
+
+docker --help >/dev/null 2>&1
+if [ $? != 0 ]; then
+  echo "===== Adding Docker Repository ====="
+  curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo apt-key add -
+  sudo add-apt-repository "deb [arch=amd64] https://download.docker.com/linux/ubuntu $(lsb_release -cs) stable"
+  sudo apt-get update
+  sudo apt-cache policy docker-ce
+
+  echo "===== Installing Docker ====="
+  sudo DEBIAN_FRONTEND=noninteractive apt-get install -y docker-ce;
+else
+  echo "Docker already installed, skipping."
+fi
+
+# Bestow docker permissions to user, required for "make test" to work
+sudo usermod -a -G docker ubuntu
+
+# Downgrade due to known issue in 1.4.6-1
```

Comment:
What issue? Is there a link?

Reply:
Link here: https://github.com/containerd/containerd/issues/5547

Reply:
It might be beneficial to embed the link within a code comment.

Reply:
Added